WNPRC_EHR/src/org/labkey/wnprc_ehr/notification/BloodDrawReviewDailyNotification.java

```diff
@@ -206,8 +206,7 @@ void getUnassignedBloodDraws() {
         void getBloodOverdraws() {
             // Creates filter.
             Date todayDate = dateToolkit.getDateToday();
-//            SimpleFilter myFilter = new SimpleFilter("date", todayDate, CompareType.DATE_GTE);  // TODO: Uncomment this when testing is finished.
-            SimpleFilter myFilter = new SimpleFilter("date", dateToolkit.getDateXDaysFromNow(-40000), CompareType.DATE_GTE);    // TODO: Delete this when testing is finished.
```

Comment:
I did read this message, but I though you mentioned when the testing for selected user finished.